streaming/src/protobuf/streaming_queue.proto

```diff
@@ -67,4 +73,34 @@ message StreamingQueueTestInitMsg {
 message StreamingQueueTestCheckStatusRspMsg {
   string test_name = 1;
   bool status = 2;
+}
+
+message StreamingQueuePullRequestMsg {
```

Comment:
redundant definition of src_actor_id, dst_actor_id, queue_id, msg_id.
How about use an unified proto?

Reply:
`MessageCommon` added.